editors/vim/README.md

```diff
@@ -74,34 +83,40 @@ let g:neoformat_try_formatprg = 1
 
 Each space in prettier options should be escaped with `\`.
 
---------------------------------------------------------------------------------
+- - -
 
 ### vim-prettier
 
 ![vim-prettier](https://raw.githubusercontent.com/prettier/vim-prettier/master/media/vim-prettier.gif?raw=true "vim-prettier")
- 
+
 #### vim-prettier - Installation
 
-Install with [vim-plug](https://github.com/junegunn/vim-plug), assumes node and yarn|npm installed globally.
+Install with [vim-plug](https://github.com/junegunn/vim-plug), assumes node and
+yarn|npm installed globally.
 
-By default it will auto format **javascript**, **typescript**, **less**, **scss** and **css** files that have "@format" annotation in the header of the file.
+By default it will auto format **javascript**, **typescript**, **less**,
+**scss** and **css** files that have "@format" annotation in the header of the
+file.
 
 ```vim
 " post install (yarn install | npm install) then load plugin only for editing supported files
-Plug 'prettier/vim-prettier', { 
-    \ 'do': 'npm install', 
-    \ 'for': ['javascript', 'typescript', 'css', 'less', 'scss'] } 
+Plug 'prettier/vim-prettier', {
+    \ 'do': 'npm install',
+    \ 'for': ['javascript', 'typescript', 'css', 'less', 'scss'] }
 ```
 
-If using other vim plugin managers or doing manual setup make sure to have `prettier` installed globally or go to your vim-prettier directory and either do `npm install` or `yarn install`
+If using other vim plugin managers or doing manual setup make sure to have
+`prettier` installed globally or go to your vim-prettier directory and either do
+`npm install` or `yarn install`
 
-When installed via vim-plug, a default prettier executable is installed inside vim-prettier.
+When installed via vim-plug, a default prettier executable is installed inside
+vim-prettier.
 
 vim-prettier executable resolution:
 
 1. Tranverse parents and search for Prettier installation inside `node_modules`
-2. Look for a global prettier installation
```

Comment:
I still think we should preserve this choice

Reply:
Should we add an option for it? or just print ordered numbers if the second number equals the first number +1?

Reply:
I'd say a good enough heuristic could be always use `1.` iff `items[1].number === "1."`.

Reply:
Opened #3024